index.html

```diff
@@ -0,0 +1,156 @@
+                <ul>
+                    <li class="activity ignite long">CC ignite</li>
+                    <li class="angagement angagement2">Get opportunities to network with CC affiliates around the world, also after the summit.</li>
+                </ul>
+            </li>
+        </ul>
+        <button id="join" class="join">Join CC Global Summit 2023</button>
+        <div id="join2">SEE THE WHOLE PROGRAM</div>
+        <hr class="hr0">
+        <br>
+        <br>
+        <br>
+    </section> 
+    <div class="featured">Featured speakers </div>
+    <br>
+    <hr class="hr1">
+    <ul id="speakers">
+
+    </ul> 
+    <button class="see-more">MORE</button>
+    <button class="see-less">See Less</button>
+    <section  id="about-section">
+        <h1 class="heading2">About The Conference <br> and The Region. </h1>
+        <p class="paragraph-2">The Global Summit brings together the community of  
+            experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+            This year, we hope to expand our invitation list including organizations and individuals.</p>
+            <p class="desk-about-par">The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+                So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together.</p>
+            <div class="contact-us">Please contact us per Email for any further questions about The Youth Submit 2023!</div>
+            <div class="email">youthsummit2023@gmail.com</div>
+    </section>
+    <div class="logo-explanation">Youth Leadership Summit 2023 Logo</div>
+         <hr class="hr1 hr21">
+         <div class="explanation">The logo of Youth Leadership Summit 2023 was decided through the logo competition from 8, June to 7, July.</div>
+         <p class="explanation2">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+            After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+            After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+         <div class="summit"><img src="images/images/about_logo 1.png" alt=""></div>
+         <div class="summit-logo"><img class="img-fluid" src="images/images/about_logo 1 (1).png" alt=""></div>
```

Comment:
As requested by the previous code reviewer

All the img tag in your code seems to be missing an alt attribute. I'll suggest you add the alt attribute with text to all your images as they are used by screen readers to know what the image is all about. 😄 👋 👍